patternlab/styleguide/source/_patterns/03-organisms/by-template/table-of-contents-overlay.twig

```diff
@@ -4,10 +4,7 @@
     {% set decorativeLink = tableOfContentsOverlay.title %}
     {% include "@atoms/decorative-link.twig" %}
     <button type="button" class="ma__toc__toc__toggle js-inline-overlay-toggle" aria-expanded="false" aria-controls="toc-overlay">
-      <span class="show">
-        +
-        <span class="ma__visually-hidden">Show table of contents</span>
-      </span>
+      <span class="show" aria-label="click to show table of contents">+</span>
```

Comment:
Capitalize the first letter in the aria-label sentence.

Reply:
It's for read-aloud only.  Capitalization doesn't matter.